commons/src/main/scala/com/lynbrookrobotics/potassium/commons/cartesianPosition/XYPosition.scala

```diff
@@ -14,26 +15,29 @@ object XYPosition {
     * @param distanceTraveled the total distance/arclength traveled
     */
   def apply(angle: Signal[Angle],
-            distanceTraveled: PeriodicSignal[Length]): PeriodicSignal[Point] = {
-    val initAngle    = angle.get
+            distanceTraveled: Signal[Length]): PeriodicSignal[Point] = {
```

Comment:
Can you make this a `SignalLike`? Then users can pass in a PeriodicSignal if that's all they have.